Demo/build.gradle

```diff
@@ -62,6 +63,11 @@ dependencies {
     implementation deps.composeRuntimeLivedata
     implementation deps.composeFoundation
     implementation deps.composeMaterial
+    implementation 'androidx.legacy:legacy-support-v4:1.0.0'
+    implementation deps.navigationFragment
+    implementation deps.navigationUI
+    implementation deps.preference
+    implementation 'androidx.preference:preference:1.1.1'
```

Comment:
Can we extract this dependency to the `deps` in the root build.gradle file?

Reply:
I think this is a duplicate dependency - removed it 👍